core/src/main/scala/filodb.core/query/QueryUtils.scala

```diff
@@ -0,0 +1,25 @@
+package filodb.core.query
```

Comment:
Kudos on moving the utility functions to a different file :) Much appreciated and thanks a tonne :)